src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Parser/SPDX30Parser.cs

```diff
@@ -147,28 +153,18 @@ public SpdxMetadata GetMetadata()
 
     public ManifestInfo[] RegisterManifest() => new ManifestInfo[] { SPDX30Constants.SPDX30ManifestInfo };
 
-    public void SetComplianceStandard(string? complianceStandardFromCli)
+    public void SetComplianceStandard(ComplianceStandardType complianceStandard)
     {
-        if (string.IsNullOrEmpty(complianceStandardFromCli))
-        {
-            // If no compliance standard is set, then we don't need to enforce any compliance standards.
-            return;
-        }
-        else if (Enum.TryParse<ComplianceStandard>(complianceStandardFromCli, true, out var complianceStandardAsEnum))
+        switch (complianceStandard.Name)
         {
-            switch (complianceStandardAsEnum)
-            {
-                case ComplianceStandard.NTIA:
-                    RequiredComplianceStandard = complianceStandardAsEnum;
-                    entititesRequiringDeserializationWithNTIA = EntitiesWithDifferentNTIARequirements;
-                    break;
-                default:
-                    break;
-            }
-        }
-        else
-        {
-            throw new ParserException($"Unable to use the given compliance standard {complianceStandardFromCli} to parse the SBOM.");
+            case "NTIA":
+                RequiredComplianceStandard = complianceStandard;
+                entititesRequiringDeserializationWithNTIA = EntitiesWithDifferentNTIARequirements;
+                break;
+            case "None":
+                break;
+            default:
```

Comment:
Best practice: Even though we don't expect any other values to get here, some future change might add that possibility. Throwing from the default case provides a level of future-proofing

Reply:
Handled https://github.com/microsoft/sbom-tool/pull/993